### PR TITLE
Add license info to current_version (fix #3964)

### LINKED ISSUE #3964: Include license.name and license.url for each addon
@kumar303 pointed out that I'm essentially doing a TONNE of effort to fetch the current version details in https://github.com/mozilla/addons-frontend/pull/1354 because the license URL and license name aren't included, because the license isn't included in the results for performance.

I imagine this is to prevent `license.text`, which could be a wall of text, from being sent over the wire. Could we include just the name and URL in the response? It would save every add-on detail page an API request, cut hundreds of lines of code from my patch, and make for speedier navigation between search results and an add-on as we'd have all data prefetched to show the content as I understand it.

### PR DESCRIPTION
**Pull request by tofumatt**
I couldn't find what tests, if any, need adding but this added the license info I need to fix #3964.

I don't want to be a pain but a quick glance seemed this would be easier than merging my huge addons-frontend patch, so if this looks okay or you can quickly tell me what I need to do to get this r+ let me know 😄 

If it's getting it your way and no one has time right now that's totally cool–just let me know and I'll leave this alone and land my add-ons frontend patch without this. cheers!

### CODE REVIEW

File: src/olympia/addons/serializers.py [hunk 1]
```diff
@@ -87,16 +87,23 @@ class Meta:
         fields = ('name', 'text', 'url')
 
 
+class CompactLicenseSerializer(LicenseSerializer):
+    class Meta:
+        model = License
+        fields = ('name', 'url')
```
Comment (EnTeQuAk) [inline src/olympia/addons/serializers.py]: I'm a fan of exposing the `id` by default too, not sure what @diox thinks about that but I found it incredibly helpful for debugging or tracing things.

**Comment by diox** [comment]
You can modify the tests that check the `current_version` in `src/olympia/addons/tests/test_serializers.py`

**Comment by diox** [comment]
> Also, given that now that license get's joined all the time anyway I wonder how much reason there is to simple SimpleVersionSerializer and VersionSerializer but that's probably not the context of this PR.

I think there is still value in keeping the serializer used for `current_version` as compact as possible. Maybe we'll have to merge them eventually, but for now it still avoids a couple other fields which makes the HTTP response smaller, so that's a small perf win.

File: src/olympia/addons/serializers.py (hunk 1 above)
Comment (tofumatt) [inline src/olympia/addons/serializers.py]: Sure, no problem with me.

**Comment by tofumatt** [comment]
Yeah, keeping the `text` attribute out of the response will cut down on a large wall of text (plus we wouldn't use it in our use case) over HTTP.

**Comment by tofumatt** [comment]
@EnTeQuAk I don't quite follow–I want the serializers to be different because I don't want to expose `license.text` to be output in the `SimpleVersionSerializer`.

I'll check out the tests, tweak the docs, and add that ID in too.

**Comment by tofumatt** [comment]
Okay, added docs and tests, let me know if it's okay or if I'm still missing something or misundertsood your comments @EnTeQuAk.

**Pull request merged**